test/lit/passes/local-cse.wast

```diff
@@ -0,0 +1,433 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
+;; NOTE: This test was ported using port_test.py and could be cleaned up.
```

Comment:
Was it? It looks like it's newly written.. Was this copied from the deleted tests?

Reply:
Yes, this is ported from the deleted ones. The filename changes (since no more flatten), and also I did a lot of cleanup and changes, so I guess git doesn't consider it a "diff" but a new file.